docs/mobile-setup.md

```diff
@@ -106,17 +43,7 @@ exports.config = {
     deviceName: 'Android Emulator',
   },
 
-  baseUrl: 'http://10.0.2.2:8000',
- 
-  // configuring wd in onPrepare
-  // wdBridge helps to bridge wd driver with other selenium clients
-  // See https://github.com/sebv/wd-bridge/blob/master/README.md
-  onPrepare: function () {
-    var wd = require('wd'),
-      protractor = require('protractor'),
-      wdBridge = require('wd-bridge')(protractor, wd);
-    wdBridge.initFromProtractor(exports.config);
-  }
+  baseUrl: 'http://10.0.2.2:8000'
```

Comment:
yeah, let's just block this on https://github.com/angular/protractor/pull/3326.  `wd-bridge` was super obnoxious to use